cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh

```diff
@@ -27,8 +27,11 @@ function up() {
 
     # remove the rancher.io kind default storageClass
     _kubectl delete sc standard
-
+    
+    export SRIOV_NODE_LABEL=${SRIOV_NODE_LABEL:-"sriov=true"}
```

Comment:
Seems like we have it all over, lets try to place it in a single place.

Reply:
I can remove the deceleration of those variable from `config_sriov.sh` and `deploy_sriov_operator.yaml` and add a check that they are initialized instead, 
what you think?